LICENSE

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2019 Maximilian Stoiber
+Copyright (c) 2020 Can Sahin
```

Comment:
I would keep the attribution to Max, since he is the original author. I think adding both makes sense.

Reply:
The only reason was he hasn't been active in this repo for years and not even responding `@mentions`. So I thought better to put someone active. Other than that I really don't care if its me or someone else :)